BPOR/BPOR.Rms/Views/Volunteer/Create.cshtml

```diff
@@ -0,0 +1,542 @@
+@model BPOR.Rms.Models.Volunteer.VolunteerFormViewModel
+
+@{
+    ViewData["Title"] = "Add a volunteer";
+    Layout = "~/Views/Shared/_Layout.cshtml";
+}
+<div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds govuk-grid-column-one-half-from-desktop">
+        <h1 class="govuk-heading-xl">Add a volunteer to the volunteer registry</h1>
+        <p class="govuk-body">All fields are mandatory unless specified otherwise</p>
+        <form asp-action="Create">
+            <div class="govuk-form-group @(Html.ViewData.ModelState["FirstName"]?.Errors.Count > 0 ? "govuk-form-group--error" : "")">
```

Comment:
Use the `FormGroupTagHelper` for error handling if possible.
Let's discuss if that needs to be extended to support more use cases.